hw11.py

```diff
@@ -0,0 +1,48 @@
+### decorated function ###
+from datetime import datetime
+def decorator(my_func):
+    def deco_func():
+        print(my_func.__name__)
+        print(datetime.now())
+        my_func()
+
+    return deco_func()

```

Comment:
Do don call this function since it will be executed immediately. User `return deco_func` instead

Reply:
You shouldn't call `deco_func` here. It should be just `return deco_func`, without parentheses